src/drive/web/modules/views/OnlyOffice/Toolbar/styles.styl

```diff
@@ -0,0 +1,9 @@
+@require 'settings/palette.styl'
```

Comment:
Why do we need to import the palette? We should use css var instead?

Reply:
Pour utiliser la fonction `rem`. Je vais regarder si on peut pas l'importer autrement du coup :)

Pour la variable css tu parles de la couleur ? Elle est spécifique pour ce séparateur, qui vient de la Bar. On peut utiliser une var style `$cozy-bar-separetor` en modifiant Ui oui

Reply:
Not sure you need to require something. Specially the palette. The `mixins` file why not, since the method is defined here it will make more sense (palette imports itself the mixins). 

Also we should really avoid to have hexa color in our app. We should rely on CSS semantic var instead. We don't have any dividerColor or something related yet?

Reply:
Oui pour `mixins` j'avais ça en tête 👍 

Non cette couleur est nulle part, c'est du spécifique cozy-bar. Et même dans cozy-bar c'est en dur en hexa.

Reply:
Idéalement il faudrait que Ui gère les divider verticaux https://github.com/cozy/cozy-ui/issues/1811

Reply:
@joel-costa un avis sur la question ?

Reply:
Oui il faut importer la palette. Le divider a une couleur dédiée dans MUI qui s'appelle palette.divider
Sur Cozy UI, cette couleur n'est ni du #D6D8Da ni du #d8d8d8 . C'est notre grey[900] à 12% d'opacité

Palette MUI : https://material-ui.com/customization/palette/
Palette Cozy UI : https://www.figma.com/file/7e3nYZkJUGZ7SLqy8xWbqx/Cozy-UI?node-id=0%3A1626
Spec divider : https://www.figma.com/file/7e3nYZkJUGZ7SLqy8xWbqx/Cozy-UI?node-id=142%3A0

Reply:
Du coup faudra changer ça dans Ui, car on est pour l'instant en `silver` cf : 
https://github.com/cozy/cozy-ui/blob/master/react/MuiCozyTheme/theme.jsx#L122
https://github.com/cozy/cozy-ui/blob/master/stylus/settings/palette.styl#L198

En revanche il y a une var `--dividerColor` que je vais pour l'instant reprendre.